JSDemos/Demos/Scheduler/BasicViews/description.md

```diff
@@ -1 +1,5 @@
-This example demonstrates four basic Scheduler views: _&laquo;Day&raquo;_, _&laquo;Week&raquo;_, _&laquo;WorkWeek&raquo;_ and _&laquo;Month&raquo;_
\ No newline at end of file
+This example demonstrates four Scheduler views: Day, Week, Work Week, and Month. You can use the View Switcher above the timetable to switch between them.
+
+To add these views to your Scheduler, list them in the [views](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array. If you do not declare it, users can select only the Day and Week views. You can also set the initially selected view in the [currentView](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#currentView) property.
```

Comment:
The first sentence somewhat implies that the array is present, I only need to add my views there. So in the second, when you refer to an array with the pronoun "it", it's a bit confusing. You never said I should declare something, and then you use "declare it". Either the first sentence needs to make it clear that they need to declare the array. Or the second sentence needs to refer to an "array" (instead of pronoun "it"). Maybe other variants are possible too. 

In the last sentence, the word "initially" is a bit vague, I'd rather use application startup. Instead of a more passive phrase "to set the initially selected view", I'd rather say "to activate a view at application startup". Lastly, the phrase "in the property" sounds weird. 

```suggestion
To add these views to your Scheduler, list them in the [views](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array. If you do not declare this array, users can switch only between the Day and Week views. To activate a specific view at application startup, use the [currentView](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#currentView) property.
```